src/server/endpoints/logoutHandler.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const HTTP_STATUSES = require('../modules/httpStatuses');
+const tokensCollection = require('../collections/tokens.js');
+const deleteAccessToken = tokensCollection.deleteDocumentByToken;
+
+function logout(req, res) {
+  res.setHeader('Content-Type', 'application/json');
+  let status;
+
+  if (req.header === null) {
```

Comment:
Header never will be null, so this condition wont be true neither.
I think we should check the authorization key instead.